src/util/errors-ts.ts

```diff
@@ -535,6 +535,16 @@ export class DeploymentPermissionDenied extends NowError<
   }
 }
 
+export class DeploymentTypeUnsupported extends NowError<'DEPLOYMENT_TYPE_UNSUPPORTED', {}> {
+  constructor() {
+    super({
+      code: 'DEPLOYMENT_TYPE_UNSUPPORTED',
+      meta: {},
+      message: `This region only accepts Serverless Docker Deployments`
```

Comment:
is that always the case with this error?

Reply:
@juliangruber I think we should say that only Now 2.0 deployments are supported?

Reply:
Oh, nevermind!